experiment/slack-event-log/handlers/subteams.go

```diff
@@ -44,7 +44,13 @@ func (h *Handler) handleSubteamUpdated(body []byte) ([]byte, error) {
 		return nil, nil
 	}
 
-	h.sendMessage("Usergroup <@%s> (%q) was *updated* by <@%s>", subteam.ID, slack.EscapeMessage(subteam.Name), subteam.UpdatedBy)
+	// This group is "modified" hourly and it is an uninteresting a noop, just filter it all.
```

Comment:
```suggestion
	// This group is "modified" hourly and is an uninteresting noop, just filter it all.
```
Thought is this the oncall alias? (which is not always a no-op, but probably should be filtered out)